drivers/cleanup.py

```diff
@@ -537,6 +537,10 @@ def __init__(self, sr, uuid, raw):
         self._vdiRef = None
         self._clearRef()
 
+    def extractUuid(path):
+        raise NotImplementedError("Implement in sub class")
+    extractUuid = staticmethod(extractUuid)
```

Comment:
Isn't it simpler to use the @staticmethod decorator and avoid using of `.__class__` below?

Reply:
I don't think that the decorator avoid the need for `__class__` and this was keeping the code consistent with the existing instance in FileVDI. But, I can try it.

Reply:
Ok, I stand corrected. That does seem to work correctly.